### PR TITLE
Created netexec docker image

### PR DESCRIPTION
**Pull request by ArtfulCoder**
The netexec binary servers hostName on http or udp.
It can also act as a proxy if other http endpoints have to be hit through itself.
It can also take ssh command requests, execute it locally and return the results on http response. 

To be used by : #13697

### CODE REVIEW

**Comment by ncdc** [comment]
@ashcrow some of this might be useful to the `kubectl exec`+proxy test work you're doing

**Comment by ncdc** [comment]
@ArtfulCoder I'm working with @ashcrow to add http/https proxy support to `kubectl exec` and `kubectl port-forward`. To do the e2e tests, we need a pod to act as a forward proxy (we're currently building a simple image using http://github.com/elazarl/goproxy), and another pod to which we can upload the locally-built kubectl, and then direct it to run `kubectl exec`. It might be worth having a quick chat to see if we can combine efforts since it looks like the code you have covers a good portion of what we've been working on.

**Comment by ashcrow** [comment]
:+1:  there looks like there is crossover between [execproxytester](https://github.com/ashcrow/kubernetes/tree/add-spdy-proxy-support/test/images/exec-proxy-tester)  and netexec.

**Comment by ArtfulCoder** [comment]
@ncd. Once this PR is merged, maybe you can just add on the features that you need ?

File: contrib/for-tests/netexec/Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.PHONY: all netexec image push clean
+
+TAG = 1.0
+PREFIX = gcr.io/google_containers
```
Comment (thockin) [inline contrib/for-tests/netexec/Makefile]: is this pushed to gcr?

File: contrib/for-tests/netexec/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM busybox
+MAINTAINER Abhishek Shah "abshah@google.com"
+
+ADD netexec netexec
```
Comment (thockin) [inline contrib/for-tests/netexec/Dockerfile]: nit: I like to add the .go file in these cases as an extra way of publishing what is happening.  Up to you.

**Comment by thockin** [comment]
LGTM, but CI failed

File: contrib/for-tests/netexec/Dockerfile (hunk 2 above)
Comment (ArtfulCoder) [inline contrib/for-tests/netexec/Dockerfile]: done

File: contrib/for-tests/netexec/Makefile (hunk 1 above)
Comment (ArtfulCoder) [inline contrib/for-tests/netexec/Makefile]: was waiting for lgtm. will do it now

**Comment by ArtfulCoder** [comment]
@thockin needs to be re-LGTMed

**Comment by ArtfulCoder** [comment]
i updated the flags.txt file to allow udp-port as a flag name

**Comment by ArtfulCoder** [comment]
pushed image.

**Pull request merged**